src/Prolens.hs

```diff
@@ -413,6 +422,28 @@ infixr 4 %~
 (%~) = over
 {-# INLINE (%~) #-}
 
+{- | 'Lens'' for a tuple on the first argument.
+
+>>> view fstL (42, "str")
+42
+
+@since 0.0.0.0
+-}
+fstL :: Lens' (a, b) a
```

Comment:
What do you think, does it make sense to make these lenses more polymorphic, so they could change the type of a pair after setting a new value? For example, go from `(Int, String)` to `(Double, String`)?
```suggestion
fstL :: Lens (a, c) (b, c) a b
```